network-config-analyzer/SchemeRunner.py

```diff
@@ -56,6 +56,15 @@ def _get_input_file(self, given_path):
             return input_file
         return given_path
 
+    def _handle_resorces_list(self, resources_list):
+        if not resources_list: # shouldn't get here
+            return None
+        if isinstance(resources_list, str):
+            resources_list = [resources_list]
+        for idx, resource in enumerate(resources_list):
+            resources_list[idx] = self._get_input_file(resource)
+        return resources_list
```

Comment:
```suggestion
        input_file_list = []
        for resource in resources_list:
            input_file_list.append(self._get_input_file(resource))
        return input_file_list
```
Better not override the list given as input